src/NodeModule.cpp

```diff
@@ -336,7 +337,11 @@ namespace
         {
             Napi::Value tmp;
             const auto params = info[0].ToObject();
-            const auto model = params.Get("model").ToString().Utf8Value();
+            const auto model = (tmp = params.Get("model"), tmp.IsUndefined() ? "" : tmp.ToString().Utf8Value());

```

Comment:
It became optional in the flux PR here https://github.com/leejet/stable-diffusion.cpp/pull/356/files#diff-a8e875e8f7684a2f9e35212d88ee9574001952eb2c049385e73d5e24fd6e2a83L172-R205 - for flux inference need to set the `diffusionModel` instead of `model`.